internal/buildengine/engine.go

```diff
@@ -373,7 +373,25 @@ func (e *Engine) Import(ctx context.Context, realmName string, moduleSch *schema
 
 // Build attempts to build all local modules.
 func (e *Engine) Build(ctx context.Context) error {
-	return errors.WithStack(e.buildWithCallback(ctx, nil))
+	schemas := make(chan *schema.Module, e.moduleMetas.Size())
+	if err := e.buildWithCallback(ctx, func(ctx context.Context, module Module, moduleSch *schema.Module, tmpDeployDir string, deployPaths []string) error {
+		schemas <- moduleSch
+		return nil
+	}); err != nil {
+		return errors.WithStack(err)
+	}
+	close(schemas)
+
+	realm := &schema.Realm{
```

Comment:
@jvmakine Can you check this realm code? I'm not sure what the engine's schema should be after a build when it comes to external realms...